Web Inspector: REGRESSION(?): Sources Tab: async stack trace divider line is not vertically centered

dcrousso opened the pull request:
#### 0dda86696f7db131ac2d3b51e91610c1282baf22
<pre>
Web Inspector: REGRESSION(?): Sources Tab: async stack trace divider line is not vertically centered
<a href="https://bugs.webkit.org/show_bug.cgi?id=242751">https://bugs.webkit.org/show_bug.cgi?id=242751</a>

Reviewed by Patrick Angle.

* Source/WebInspectorUI/UserInterface/Views/CallFrameTreeElement.css:
(.tree-outline .item.call-frame.async-boundary::before,):

Canonical link: <a href="https://commits.webkit.org/252468@main">https://commits.webkit.org/252468@main</a>
</pre>

patrickangle reviewed (approved):
A time honored CSS tradition of `vertical-align` not doing what anyone wants 😅

dcrousso committed:
Web Inspector: REGRESSION(?): Sources Tab: async stack trace divider line is not vertically centered
https://bugs.webkit.org/show_bug.cgi?id=242751

Reviewed by Patrick Angle.

* Source/WebInspectorUI/UserInterface/Views/CallFrameTreeElement.css:
(.tree-outline .item.call-frame.async-boundary::before,):

Canonical link: https://commits.webkit.org/252468@main
